src/ordered_set_lang.ml

```diff
@@ -291,31 +297,40 @@ module Unexpanded = struct
     in
     loop t.ast Pos init
 
-  let expand t ~files_contents ~f  =
+  let expand t ~dir ~files_contents ~(f : String_with_vars.t -> Value.t list) =
     let context = t.context in
+    let f_elems s =
+      let loc = String_with_vars.loc s in
+      List.map ~f:(fun s -> (loc, Value.to_string ~dir s)) (f s)
+      |> Ast.of_list
+    in
     let rec expand (t : ast) : ast_expanded =
       let open Ast in
       match t with
-      | Element s -> Element (String_with_vars.loc s, f s)
+      | Element s -> f_elems s
       | Standard -> Standard
       | Include fn ->
         let sexp =
-          let fn = f fn in
-          match String.Map.find files_contents fn with
+          let path =
+            match f fn with
+            | [x] -> Value.to_path ~dir x
+            | _ ->
+              Exn.code_error "Ordered_set_lang.Unexpanded.expand path"
```

Comment:
I believe we can reach this case with `(:include %{cc})`, so it shouldn't be a code error

Reply:
Ah ok, I see. Will do that.